roles/edpm_ovn/tasks/run.yml

```diff
@@ -43,9 +43,52 @@
       ovn_controller: "{{ lookup('template', 'kolla_ovn_controller.yaml.j2') | from_yaml }}"
   when: not edpm_use_bootc
 
-- name: Run ovn_controller container
-  ansible.builtin.systemd_service:
-    name: edpm-compute@ovn_controller
-    state: started
+- name: Configure ovn_controller for bootc deployments
+  block:
+  - name: Collect default pod spec
+    ansible.builtin.slurp:
+      src: /usr/share/containers/systemd/ovn_controller.yaml
+    register: ovn_controller_pod_spec
+
+  - name: Decode and parse the YAML content
+    ansible.builtin.set_fact:
+      ovn_controller_pod_spec: "{{ ovn_controller_pod_spec.content | b64decode | from_yaml }}"
+
+  - name: Update fields in the YAML data
+    ansible.builtin.set_fact:
+      updated_pod_spec: >-
+        {{
+          ovn_controller_pod_spec | combine({
+            'spec': {
+              'containers': ovn_controller_pod_spec.spec.containers | zip_longest([], [{'image': edpm_ovn_controller_agent_image}]) | map('combine') | list,
```

Comment:
I believe that if we were to customize the image like this then by definition the container is no longer "logically bound". Instead, it would be considered a "floating" container per https://containers.github.io/bootc/logically-bound-images.html#comparison-with-default-podman-systemd-units

and also:

> There is no mechanism to inject arbitrary arguments to the podman pull (or equivalent) invocation used by bootc.

which seems to apply additional mounts or other options passed to podman pull are not possible.

The dynamically-injected ConfigMaps[1][2] may provide some customization, but that is still not likely for the app container image itself, b/c once that is changed to some other image, then that no longer fits into how logically bound images should be managed with the lifecycle of the base bootc image itself.

Point being...if we choose to allow the ability to podman run any arbitrary image at runtime, then these really aren't logically bound images at all, but are considered "floating".

The question becomes, should we adopt logically bound images, and require our end users to be building new bootc images before deploying EDPM nodes, depending on if they need to customize any of the container images? We could ship a bootc image that had all the images logically bound, but if a user wanted to run a different one (from a partner, etc) they they would need to rebuild that image.

I do like the quadlet/systemd design, and I think we can still adopt that.

[1] https://containers.github.io/bootc/building/guidance.html?highlight=configmap#configuration
[2] https://github.com/containers/bootc/issues/22